wfl/App_Start/WebApiConfig.cs

```diff
@@ -1,17 +1,21 @@
 ﻿using System.Web.Http;
+using Drum;
+using Hypermedia;
+using Hypermedia.Transforms;
+using Hypermedia.Siren;
+using Symphono.Wfl.Models;
 
 namespace Symphono.Wfl
 {
     public static class WebApiConfig
     {
         public static void Register(HttpConfiguration config)
         {
-            config.MapHttpAttributeRoutes();
+            config.EnableHypermedia()
+                .RegisterDefaultTransform(new ReflectiveTransform());
```

Comment:
I haven't looked at the actual Hypermedia.NET project in a while, but what does the `ReflectiveTransform()` do for us here?